functions/tests/end_to_end_test.go

```diff
@@ -38,6 +38,8 @@ var skipTests = map[string]string{
 	"covariance_missing_column_2": "need to support known errors in new test framework (https://github.com/influxdata/flux/issues/536)",
 	"drop_before_rename":          "need to support known errors in new test framework (https://github.com/influxdata/flux/issues/536)",
 	"drop_referenced":             "need to support known errors in new test framework (https://github.com/influxdata/flux/issues/536)",
+	"drop_non_existent":           "need to support known errors in new test framework (https://github.com/influxdata/flux/issues/536)",
```

Comment:
Which files do you refer to?
I can only find `integral.in.csv` and `integral_columns.in.csv`, but those are not relevant for this PR.

Reply:
Ah, I'm wrong about the csv files. What about these two skipped tests? I see the `.flux` files for them below.

Reply:
Yeah, they must be skipped because the support for checking errors has been dropped with the new testing framework.

They are ready for when that issue will be fixed.

Reply:
I created them and later I discovered they couldn't be executed. Anyways, they will be valuable after the fix.

Reply:
Ah, I understand now.

Reply:
@jsternberg created a PR for integral tests here #683